cmux.go

```diff
@@ -177,7 +177,8 @@ func (l muxListener) Accept() (net.Conn, error) {
 // MuxConn wraps a net.Conn and provides transparent sniffing of connection data.
 type MuxConn struct {
 	net.Conn
-	buf buffer
+	buf     bytes.Buffer
```

Comment:
Yes, I do. I also object to "optimizations" like https://github.com/soheilhy/cmux/commit/04d6a17e41#diff-c5f7bf6980b6b3b699ddc715cd7e7f7dR28 which have been sources of bugs in the past.